docs/website/references/cli/defradb_client_document_get.md

```
@@ -7,17 +7,17 @@ View document fields.
 View document fields.
 
 ```
-defradb client collection get [-i --identity] [--show-deleted] <docID>  [flags]
+defradb client document get [-i --identity] [--show-deleted] <docID>  [flags]
 ```
 
 ### Examples
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)